src/Web/Sfa.Das.Sas.Shared.Components/ViewComponents/ApprenticeshipDetails/ApprenticeshipDetailsViewComponent.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Threading.Tasks;
```

Comment:
Was it a conscious decisions to put this component in it's own folder and not in the 'fat' one with the search components?

Reply:
yes, another application may want to show apprenticeship details which arent linked to FAT so thought this would make a clear distinction